__test__/backend/schema/schemaService.test.ts

```
@@ -25,6 +25,7 @@ describe('SchemaService Unit Tests', () => {
     name: 'Test Schema',
     description: 'Test Description',
     schemaText: 'CREATE TABLE test (id SERIAL PRIMARY KEY, name TEXT);',
+    serviceId: 'db9caeca-aa1a-46f6-84de-adfe0a414c03',
```

Biome (1.9.4)

[error] This property value named serviceId is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named serviceId


(lint/suspicious/noDuplicateObjectKeys)